### ISSUE TITLE
Algorithm ExtraLaunch Parameters

### ISSUE THREAD

**Issue by matmanmining** [nicehash/NiceHashMiner#906]
Hello.  I am using NHML 1.8.6.

I am using claymore dual algorithm and would like to specify the -dcri variable for each of my six gpus, since they are of all different makes and models.

I have attempted to set the -dcri variable for each card in the ExtraLaunch Parameters tab. but these do not appear to stick.  I have gone to the start.bat file and modified the bat to show -dcri 32 which when I use the start.bat file manually, it works.  But if I use NHLM, the parameter does not stick.  Can anyone give me some insight into this issue?

Thanks.

**Comment by matmanmining** [comment]
So I tried assigning the startup parameters using the beta version, which appears to be an older version.  Could this be a bug in the 1.8.1.6 release?  Maybe even specific to Claymore dual?

Although it works in the older beta version, the algorithms are not the most recent and therefor less efficient, I suspect.  I hope some one of the coders can comment.  Thanks.

Edit:  As I suspected, my hashrate using NHLM 1.8.2.0 Beta (v10.0) is slightly lower than with Claymore (v10.3).  I did discover the tuning mode for dual on the beta version.  I really like it.  Just wish the miners were updated to the most recent version.
